lib/logitech_receiver/settings_templates.py

```diff
@@ -403,14 +442,18 @@ def check_feature(name, featureId, field_name=None):
 		except Exception as reason:
 			_log.error("check_feature[%s] inconsistent feature %s", featureId, reason)
 
-	check_feature(_HI_RES_SCROLL[0], _F.HI_RES_SCROLLING)
-	check_feature(_LOW_RES_SCROLL[0], _F.LOWRES_WHEEL)
-	check_feature(_HIRES_INV[0],     _F.HIRES_WHEEL, "hires_smooth_invert")
-	check_feature(_HIRES_RES[0],     _F.HIRES_WHEEL, "hires_smooth_resolution")
-	check_feature(_FN_SWAP[0],       _F.FN_INVERSION)
-	check_feature(_FN_SWAP[0],       _F.NEW_FN_INVERSION, 'new_fn_swap')
-	check_feature(_FN_SWAP[0],       _F.K375S_FN_INVERSION, 'k375s_fn_swap')
-	check_feature(_DPI[0],           _F.ADJUSTABLE_DPI)
-	check_feature(_POINTER_SPEED[0], _F.POINTER_SPEED)
-	check_feature(_SMART_SHIFT[0],   _F.SMART_SHIFT)
-	check_feature(_BACKLIGHT[0],   	 _F.BACKLIGHT2)
+	check_feature(_HI_RES_SCROLL[0],       _F.HI_RES_SCROLLING)
+	check_feature(_LOW_RES_SCROLL[0],      _F.LOWRES_WHEEL)
+	check_feature(_HIRES_INV[0],           _F.HIRES_WHEEL, "hires_smooth_invert")
+	check_feature(_HIRES_RES[0],           _F.HIRES_WHEEL, "hires_smooth_resolution")
+	check_feature(_FN_SWAP[0],             _F.FN_INVERSION)
+	check_feature(_FN_SWAP[0],             _F.NEW_FN_INVERSION, 'new_fn_swap')
+	check_feature(_FN_SWAP[0],             _F.K375S_FN_INVERSION, 'k375s_fn_swap')
+	check_feature(_DISABLE_CAPS_LOCK[0],   _F.KEYBOARD_DISABLE_KEYS, 'disable_caps_lock')
+	check_feature(_DISABLE_SCROLL_LOCK[0], _F.KEYBOARD_DISABLE_KEYS, 'disable_scroll_lock')
+	check_feature(_DISABLE_INSERT[0],      _F.KEYBOARD_DISABLE_KEYS, 'disable_insert')
+	check_feature(_DISABLE_WIN[0],         _F.KEYBOARD_DISABLE_KEYS, 'disable_win')
```

Comment:
Why are these all KEYBOARD_DISABLE_KEYS?  Shouldn't they be the individual key disables?